_maps/map_files/RandomRuins/SpaceRuins/freighter.dmm

```diff
@@ -0,0 +1,1261 @@
+/obj/structure/rack,
+/obj/item/clothing/head/kitty,
+/obj/item/mecha_modkit/voice/honk{
+	pixel_y = 10;
+	pixel_x = 3
+	},
+/turf/simulated/floor/mineral/plastitanium,
+/area/ruin/space/powered)
+"Pf" = (
+/obj/machinery/computer/nonfunctional{
+	dir = 8
+	},
+/turf/simulated/floor/plasteel{
+	icon_state = "dark"
+	},
+/area/ruin/space/powered)
+"Pn" = (
+/obj/machinery/suit_storage_unit/standard_unit,
+/obj/machinery/light/small,
+/turf/simulated/floor/mineral/plastitanium,
+/area/ruin/space/powered)
+"PE" = (
+/obj/structure/bookcase/random,
+/turf/simulated/floor/plasteel{
+	dir = 10;
+	icon_state = "caution"
+	},
+/area/ruin/space/powered)
+"Qe" = (
+/obj/effect/turf_decal/delivery/white/partial{
+	dir = 8
+	},
+/obj/effect/turf_decal/delivery/white/partial{
+	dir = 4
+	},
+/turf/simulated/floor/mineral/plastitanium,
+/area/ruin/space/powered)
+"QM" = (
+/obj/structure/railing,
+/obj/item/flag/ussp,
+/turf/simulated/floor/plasteel{
+	dir = 5;
+	icon_state = "caution"
+	},
+/area/ruin/space/powered)
+"Sp" = (
+/obj/structure/largecrate,
+/turf/simulated/floor/mineral/plastitanium,
+/area/ruin/space/powered)
+"SL" = (
+/obj/structure/railing,
+/mob/living/simple_animal/hostile/russiannian,
+/turf/simulated/floor/plasteel,
+/area/ruin/space/powered)
+"Tf" = (
+/obj/machinery/economy/vending/snack/free,
+/turf/simulated/floor/plasteel,
+/area/ruin/space/powered)
+"TW" = (
+/obj/effect/turf_decal/delivery/white/partial{
+	dir = 8
+	},
+/turf/simulated/floor/mineral/plastitanium,
+/area/ruin/space/powered)
+"UZ" = (
+/mob/living/simple_animal/hostile/russiannian,
+/turf/simulated/floor/mineral/plastitanium,
+/area/ruin/space/powered)
+"Wb" = (
+/mob/living/simple_animal/hostile/russiannian,
+/turf/simulated/floor/plasteel{
+	icon_state = "cafeteria"
+	},
+/area/ruin/space/powered)
+"WF" = (
+/obj/machinery/door/airlock/external,
+/obj/structure/fans/tiny,
+/turf/simulated/floor/pod/dark,
+/area/ruin/space/powered)
+"WR" = (
+/obj/structure/chair/comfy/shuttle{
+	dir = 4
+	},
+/obj/effect/spawner/random_spawners/blood_maybe,
+/turf/simulated/floor/plasteel{
+	icon_state = "dark"
+	},
+/area/ruin/space/powered)
+"XB" = (
+/obj/structure/table,
+/obj/item/paper{
+	info = "<h3>Voidhopper of Nexus Shipping Manifest</h3><hr><br>Order: #7934<br>Destination: <i> The destination seems to have been blacked out.</i><br>Requested By: Athena, Praetorian of Gliese<br>CONTENTS:<br><ul><li>1 pair of kitty ears</li><li>1 banhammer</li><li>1 exosuit mining scanner</li><li>1 diamond-tipped exosuit drill</li><li>1 "LittleBoy" Mining Grenade Launcher</li><li>1 mecha voice modification kit: Honk</li><li>20 plastitanium glass</li><li>50 uranium sheets</li><li>50 reinforced plasma glass</li><li>50 metal sheets</li><li>1 fuel tank</li><li>1 radiant dance machine mark IV</li><li>1 soap</li><li>1 advanced mop</li><li>1 advanced space cleaner</li></ul><br>CHECK CONTENTS AND SIGN BELOW THE LINE TO CONFIRM RECEIPT OF GOODS<hr><b>X: <i> Athena, Praetorian of Gliese </i> </b>";
```

Comment:
create a file `/code/modules/away_missions/mission_code/ruins/nian_freighter.dm` or similar and put this in there as a paper subtype:

```c
/obj/item/paper/fluff/ruins/nian_freighter/manifest
	name = "Voidhopper of Nexus Shipping Manifest"
	info = {"
		<h3>Voidhopper of Nexus Shipping Manifest</h3>
		<hr />
		<br />Order: #7934<br />Destination:
		<i> The destination seems to have been blacked out.</i><br />Requested By:
		Athena, Praetorian of Gliese<br />CONTENTS:<br />
		<ul>
		  <li>1 pair of kitty ears</li>
		  <li>1 banhammer</li>
		  <li>1 exosuit mining scanner</li>
		  <li>1 diamond-tipped exosuit drill</li>
		  <li>1 "LittleBoy" Mining Grenade Launcher</li>
		  <li>1 mecha voice modification kit: Honk</li>
		  <li>20 plastitanium glass</li>
		  <li>50 uranium sheets</li>
		  <li>50 reinforced plasma glass</li>
		  <li>50 metal sheets</li>
		  <li>1 fuel tank</li>
		  <li>1 radiant dance machine mark IV</li>
		  <li>1 soap</li>
		  <li>1 advanced mop</li>
		  <li>1 advanced space cleaner</li>
		</ul>
		<br />CHECK CONTENTS AND SIGN BELOW THE LINE TO CONFIRM RECEIPT OF GOODS
		<hr />
		<b>X: <i> Athena, Praetorian of Gliese </i> </b>
	"}
```

See the similar ruin papers in `code\modules\awaymissions\mission_code\ruins\deepstorage.dm` as examples.

Reply:
Alright, could you check if I did what you said to do properly? Thanks!